samples/testapp/src/commonMain/kotlin/com/android/identity/testapp/ui/QrCodesScreen.kt

```diff
@@ -0,0 +1,89 @@
+package com.android.identity.testapp.ui
+
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextButton
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import com.android.identity.appsupport.ui.qrcode.ShowQrCodeDialog
+import com.android.identity.appsupport.ui.qrcode.ScanQrCodeDialog
+
+@Composable
+fun QrCodesScreen(
+    showToast: (message: String) -> Unit
+) {
+    val showMdocQrCodeDialog = remember { mutableStateOf(false) }
+    val showUrlQrCodeDialog = remember { mutableStateOf(false) }
+    val showQrScanDialog = remember { mutableStateOf(false) }
+
+    if (showMdocQrCodeDialog.value) {
+        ShowQrCodeDialog(
+            title = "Scan code on reader",
+            description = "Your personal information won't be shared yet. You don't need to hand your phone to anyone to share your ID.",
+            dismissButton = "Close",
+            data = "mdoc:owBjMS4wAYIB2BhYS6QBAiABIVggWojRgrzl9C76WZQ/MzWdAuipaP8onZPl" +
```

Comment:
Add a comment explaining what this magic value is?

Reply:
OK, adding this: _This is the DeviceEngagement test vector from ISO/IEC 18013-5:2021 Annex D encoded as specified in clause 8.2.2.3_